es-ds-components/components/es-collapse.vue

```diff
@@ -85,3 +85,12 @@ const onClick = ({ value }: { value: boolean }) => {
         <slot />
     </panel>
 </template>
+<style lang="scss" scoped>
+@use '@energysage/es-ds-styles/scss/variables' as variables;
+
+:deep(.es-collapse-toggler:focus-visible) {
+    outline: 2px auto #2e46b9;
```

Comment:
Since we're importing variables above already, probably makes sense to use `variables.$blue-600` here instead of the hex.